### PR TITLE
Update main.py

### PR DESCRIPTION
**Pull request by dhruvsheth-ai**
updated Lib to 2.0.0.1
Tested on armv7l

### CODE REVIEW

**Comment by Luxonis-Brandon** [comment]
Thanks!

File: gen2-human-pose/main.py [hunk 1]
```diff
@@ -167,6 +169,12 @@ def get_frame():
 
             if cv2.waitKey(1) == ord('q'):
                 break
+            elif cv2.waitKey(2) == ord('t'):
```
Comment (szabi-luxonis) [inline gen2-human-pose/main.py:172]: `cv2.waitKey(X) ` [waits](https://docs.opencv.org/2.4/modules/highgui/doc/user_interface.html#waitkey) for X milliseconds for keypress. 
Change to: 
```
key = cv2.waitKey(1)
if key == ord('q'):
     break
elif key == ord('t'):
```

**Review by szabi-luxonis — CHANGES_REQUESTED** [review]

File: gen2-human-pose/main.py (hunk 1 above)
Comment (szabi-luxonis) [reply gen2-human-pose/main.py:172]: And thanks for the change. Otherwise LGTM

Comment (dhruvsheth-ai) [reply gen2-human-pose/main.py]: Thanks, changed the file

**Review by szabi-luxonis — APPROVED** [review]
LGTM

**Pull request merged**